lib/app.ts

```diff
@@ -0,0 +1,45 @@
+const options = require("./options");
```

Comment:
Is there a reason these still using require?

Reply:
nope

Reply:
actually yes, importing that old style node modules thing is problematic - so I have left all old JS imports as require, they can be changed when tsified